content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -0,0 +1,465 @@
+---
+layout: bt_wiki
+title: Helm 3 Plugin
+category: Official Plugins
+draft: false
+weight: 100
+aliases: ["/plugins/helm/", "/developer/official_plugins/helm/"]
+---
+
+Helm is the first application package manager running on top of Kubernetes. It allows describing the application structure through convenient helm-charts and managing it with simple commands.
+Helm uses three main concepts:
+
+* Charts
+
+* Repository
+
+* Release
+
+A Chart is a Helm package. It contains all of the resource definitions necessary to run an application, tool, or service inside of a Kubernetes cluster. Think of it like the Kubernetes equivalent of a Homebrew formula, an Apt pkg, or a Yum RPM file.
+
+A Repository is a place where charts can be collected and shared. It's like Perl's CPAN archive or the Fedora package Database, but for Kubernetes packages.
+
+A Release is an instance of a chart running in a Kubernetes cluster. One chart can often be installed many times into the same cluster. And each time it is installed, a new release is created. Consider a MySQL chart. If you want two databases running in your cluster, you can install that chart twice. Each one will have its own release, which will in turn have its own release name.
```

Comment:
I'm not sure we need to explain helm terminology here. Ask @ofercloudify and @natishalom if they disagree. Let's hope someone using the plugin at least knows how to use helm.

Reply:
Normally I would agree with Trammel.
In this specific case I see no harm in having it so I'll keep it.